src/Handler.php

```diff
@@ -12,24 +12,85 @@ class Handler
     protected $client;
 
     /**
+     * The previously registered error handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousErrorHandler;
+
+    /**
+     * The previously registered exception handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousExceptionHandler;
+
+    /**
      * Register our exception handler.
      *
      * @param \Bugsnag\Client|string|null $client client instance or api key
+     * @param bool $callPrevious whether or not to call the previous handlers
      *
      * @return static
      */
-    public static function register($client = null)
+    public static function register($client = null, $callPrevious = true)
     {
         $handler = new static($client instanceof Client ? $client : Client::make($client));
 
-        set_error_handler([$handler, 'errorHandler']);
-        set_exception_handler([$handler, 'exceptionHandler']);
-        register_shutdown_function([$handler, 'shutdownHandler']);
+        $handler->registerErrorHandler($callPrevious);
+        $handler->registerExceptionHandler($callPrevious);
+        $handler->registerShutdownHandler();
 
         return $handler;
     }
 
     /**
+     * Register the bugsnag error handler and save the previous one
+     * (if it exists) to call later.
+     *
+     * @param bool $callPrevious whether or not to call the previous handler
+     *
+     * @return void
+     */
+    public function registerErrorHandler($callPrevious = true)
+    {
+        $previous = set_error_handler([$this, 'errorHandler']);
+
+        if ($callPrevious) {
+            $this->previousErrorHandler = $previous;
+        }
+    }
+
+    /**
+     * Register the bugsnag exception handler and save the previous one
+     * (if it exists) to call later.
```

Comment:
Please ensure that "short descriptions" are less than 8 chars and don't overflow past one line.